config/config.go

```diff
@@ -299,6 +300,10 @@ type GlobalConfig struct {
 	ScrapeTimeout model.Duration `yaml:"scrape_timeout,omitempty"`
 	// How frequently to evaluate rules by default.
 	EvaluationInterval model.Duration `yaml:"evaluation_interval,omitempty"`
+	// Duration after which PromQL queries are logged.
+	QueryLogMinDuration *model.Duration `yaml:"query_log_min_duration,omitempty"`
```

Comment:
I think we'd said just to log everything, in any case "Min" in the name could be confused to mean minutes.

Reply:
Removed duration.